tests/test_cli.py

```diff
@@ -347,11 +347,12 @@ def test_lambda_fold(self):
         runner = CliRunner()
         with temporary_file() as output_filename:
             with open(output_filename, 'wt') as output:
+                schema_path = os.path.join(TESTDATA, 'lambda_fold_schema.json')
```

Comment:
More of an FYI but [pathlib](https://docs.python.org/3.7/library/pathlib.html) is nice

Reply:
Thanks Brian I will have a look and potentially use it in the future!